contracts/merger/PegExchangerDripper.sol

```diff
@@ -0,0 +1,28 @@
+pragma solidity ^0.8.4;
+
+import "./PegExchanger.sol";
+
+contract PegExchangerDripper {
+
+    IERC20 public constant TRIBE = IERC20(0xc7283b66Eb1EB5FB86327f08e1B5816b0720212B);
+    address public constant CORE = 0x8d5ED43dCa8C2F7dFB20CF7b53CC7E593635d7b9;
+    address public constant PEG_EXCHANGER = 0xc09BB5ECf865e6f69Fe62A43c27f036A426909f7;
+    
+    uint256 public constant THRESHOLD = 5e18; // 5M TRIBE cutoff for dripping
+    uint256 public constant DRIP_AMOUNT = 20e18; // drip 20M TRIBE
+ 
+    function drip() external {
+        require(!PegExchanger(PEG_EXCHANGER).isExpired(), "expired"); // ensure pegExchanger live
+        require(TRIBE.balanceOf(PEG_EXCHANGER) <= THRESHOLD, "over threshold"); // ensure under drip threshold
```

Comment:
You can simplify here and just require `isEligible`

Reply:
Do you think having separate error messages is useful?

I *think* inline is more gas efficient but I could be full of shit

Reply:
it technically is more gas efficient because you reduce the jump call you have to make to call `isEligible`. this isn't a sticking point so if you'd prefer it this way we can just leave as is